components/collective-page/sections/Contribute.js

```diff
@@ -219,24 +205,19 @@ class SectionContribute extends React.PureComponent {
 
   render() {
     const { collective, tiers, events, connectedCollectives, contributors, isAdmin } = this.props;
-    const { draggingContributionsOrder, isSaving, showTiersAdmin } = this.state;
-    const orderKeys = draggingContributionsOrder || getCollectiveContributionCardsOrder(collective);
-    const sortedTiers = this.getSortedCollectiveTiers(tiers, orderKeys);
+    const { isSaving, showTiersAdmin } = this.state;
     const isEvent = collective.type === CollectiveType.EVENT;
     const isProject = collective.type === CollectiveType.PROJECT;
     const isFund = collective.type === CollectiveType.FUND;
-    const hasCustomContribution = !get(collective, 'settings.disableCustomContributions', false);
-    const hasCryptoContribution =
-      !get(collective, 'settings.disableCryptoContributions', true) &&
-      get(collective, 'host.settings.cryptoEnabled', false);
-    const hasContribute =
-      isAdmin || (collective.isActive && (sortedTiers.length || hasCustomContribution || hasCryptoContribution));
```

Comment:
@gustavlrsn Just noticing this, why did we change the definition for `hasContribute` here?